ci-operator/config/openshift/openshift-tests-private/README.md

```diff
@@ -60,31 +60,44 @@ NOTE: We can use below script to generate cron settings against the frequency:
       ci-operator/config/openshift/openshift-tests-private/tools/generate-cron-entry.sh
 ~~~
 
-# Installer (base_images)
+# images
```

Comment:
The `images` stuff and `releases` stuff are not specific for openshift-tests-private, I would suggest remove them out from this README, and perhaps add them directly in a google doc.  
Also you can share your google doc to the beginners directly without linking it here.  Someone read this doc and unable to access linked google doc can get confused.